src/js/interface.js

```diff
@@ -350,6 +350,17 @@ window.addEventListener("load", function() {
         dmaDisplayMemoryContainer.style.display = "inline-block";
     }
 
+    function getColorFromMem(x) {
+        var blue = Math.round((x & 0x1f) / 31.0 * 255.0);
+        var green = Math.round(((x >> 5) & 0x1f) / 31.0 * 255.0);
+        var red = Math.round(((x >> 10) & 0x1f) / 31.0 * 255.0);
+        var redString = ("0"+red.toString(16)).slice(-2);
+        var greenString = ("0"+green.toString(16)).slice(-2);
+        var blueString = ("0"+blue.toString(16)).slice(-2);
+        var color = "#"+redString+greenString+blueString;
```

Comment:
Looks correct (but of course I thought it was correct before - forgot about the leading 0s), although maybe it would be easier to just use the RGB values directly with
```js
var color = `rgb(${red}, ${green}, ${blue})`;
```
And do away with the hex conversion